src/python/grpcio/tests/unit/_cython/cygrpc_test.py

```diff
@@ -188,12 +193,12 @@ def testEcho(self):
                          CLIENT_METADATA_ASCII_VALUE),
         cygrpc.Metadatum(CLIENT_METADATA_BIN_KEY, CLIENT_METADATA_BIN_VALUE)])
     client_start_batch_result = client_call.start_batch(cygrpc.Operations([
-        cygrpc.operation_send_initial_metadata(client_initial_metadata),
-        cygrpc.operation_send_message(REQUEST),
-        cygrpc.operation_send_close_from_client(),
-        cygrpc.operation_receive_initial_metadata(),
-        cygrpc.operation_receive_message(),
-        cygrpc.operation_receive_status_on_client()
+        cygrpc.operation_send_initial_metadata(client_initial_metadata, 0),
```

Comment:
I'm not yet ready to suggest that we include such a constant in our public API, but at least within this file how about an `_EMPTY_FLAGS` constant? All these zeroes start to melt one's brain appearing as they do without context.